src/ethereum/spurious_dragon/vm/interpreter.py

```diff
@@ -113,12 +115,14 @@ def process_create_message(message: Message, env: Environment) -> Evm:
     # take snapshot of state before processing the message
     begin_transaction(env.state)
 
+    increment_nonce(env.state, message.current_target)
```

Comment:
I was expecting this to be added to the EIP 161 commit and not 170. 
I'll try to clean this up in the PR that I'm working on.

Reply:
Oops